src/DssSpell.t.base.sol

```diff
@@ -89,6 +89,7 @@ contract DssSpellTestBase is DSTest, DSMath {
         uint256 calc_tau;
         uint256 calc_step;
         uint256 calc_cut;
+        bool    offboarding; // if setting to true, we need to remove these  when we are done with the lerp
```

Comment:
we could consider adding another flag (eg. `live` or `active`) to keep track of all offboarded ilks as this flag doesn't include other offboarded ilks (eg. [USDC-B](https://beta.mcdstate.info/collateral/USDC-B), and so on), but only lerped ones to be used in the `mat` range check to replace the ilks block, and consider adding ad-hoc tests to check for active vaults or similar to assess that the ilk is live (DC > 0, Autoline enabled and active vaults) or not (DC = 0, Autoline disabled and no active vaults).
cc: @godsflaw @gbalabasquer

Reply:
We could open a separate PR to discuss about this along with ad-hoc test coverage to check for active/offboarded ilks.

Reply:
#211

Reply:
started discussion there